app/views/admin/courses/edit.html.erb

```diff
@@ -0,0 +1,2 @@
+<% content_for :title, "#{t(:edit)} #{Course.model_name.human}: #{@current_course.name}" %>
+<%= render partial: 'form', formats: [:html], handlers: [:erb] %>
```

Comment:
You don't need the `formats` and `handlers` option here (I think by default Rails will use the same as the current file).